### ISSUE TITLE
[Keyboard Navigation-Home Page-Navigation]Multiple H1 tags are used on Home Page. 

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2598]
**User Experience:**
Users who depends Keyboard and screen reader will difficulty to access the headings if they are not defined properly under Heading tags.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Browser : Edge.
Tool Used: Edge Developer Tool

**Repro Steps:-**
1. Navigate to url : https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3. Verify that all the Headings are defined according to proper heading structure

**Actual Result:-**
All the headings like 'Microsoft Make Code,Hands-on computing education,About Microsoft MakeCode....etc' showing on Home Page are defined under H1 tags

**Expected Result:-**
Multiple H1 tags should not be used on Home Page .Heading structure should be is proper order like H1>H2>H3>H4>H5.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Suggested fix:**
In all the applicable forms we need to check below-
1-Headings in HTML range from H1 (the most important) to H6 (the least important). It’s best to reserve Heading1 (H1) on a web page for the title of that page.  
2-Headings don’t need to decrease from 1 to 6 on every page. While you don’t need to use all of the headings, they should still be sequential.  
For example, avoid skips from H2 to H4. 
3-If you regularly produce similar content, keep headings consistent

**Please refer the Screen shot:**
![mas2 4 6_home page_multiple h1 tags are used](https://user-images.githubusercontent.com/30434417/28814312-adc5214c-76ba-11e7-83d6-f6bc8dfd2a37.jpg)

**Comment by microsoftsam** [comment]
Fixed on makecode.com

**Comment by v-mesala** [comment]
Bug is verified and fixed .hence closed the bug.
